components/home/Hero.vue

```diff
@@ -29,19 +22,15 @@
             <div>✓ &nbsp; Unlimited forms</div>
           </div>
         </div>
-        <div
-          class="
-            col-lg-6
-            d-flex
-            align-items-center
-            justify-content-center
-          "
-        >
-          <img
+        <div class="col-lg-6 d-flex align-items-center justify-content-center">
+          <!-- <img
             src="@/assets/images/home/hero.svg"
             alt="Formester form building with drag and drop interface"
             class="img-fluid hero__image"
-          />
+          /> -->
+          <video loop muted autoplay playsinline style="max-width: 130%">
+            <source src="@/assets/video/hero.webm" type="video/webm" />
```

Comment:
update the folder name to `videos`
then the updated path would look like `@assets/videos/hero.webm`

Reply:
For the video @ankitsinghaniyaz will comment